api.go

```diff
@@ -287,18 +287,25 @@ func apiSetDeploymentResults(w http.ResponseWriter, r *http.Request) {
 	w.Write([]byte("OK"))
 }
 
+
+func addHeaders(req *http.Request) {
+	var token = services.Config().GetString("apigeesync_bearer_token")
+	req.Header.Add("Authorization", "Bearer "+token)
+}
+
+
 func transmitDeploymentResultsToServer(validResults apiDeploymentResults) error {
 
 	retryIn := bundleRetryDelay
 	maxBackOff := 5 * time.Minute
 	backOffFunc := createBackoff(retryIn, maxBackOff)
 
-	uri, err := url.Parse(apiServerBaseURI.String())
+	_, err := url.Parse(apiServerBaseURI.String())
```

Comment:
Yes, testing it. Good idea before constructing url :) to prevent from FS/FTP attacks